src/Symfony/Bridge/Twig/Extension/YamlExtension.php

```diff
@@ -39,9 +40,16 @@ public function encode($input, $inline = 0, $dumpObjects = 0)
             $dumper = new YamlDumper();
         }
 
-        if (defined('Symfony\Component\Yaml\Yaml::DUMP_OBJECT') && is_bool($dumpObjects)) {
-            @trigger_error('Passing a boolean flag to toggle object support is deprecated since version 3.1 and will be removed in 4.0. Use the Yaml::DUMP_OBJECT flag instead.', E_USER_DEPRECATED);
-            $dumpObjects = (int) $dumpObjects;
```

Comment:
As we appplied this change in 2.x branches too (to avoid deprecation warnings when using highest deps), this should be changed there too.